File: app/src/main/java/fergaral/datetophoto/utils/Folders.java
package fergaral.datetophoto.utils;

import java.io.File;
import java.util.HashMap;
import java.util.Map;

/**
 * Created by fer on 7/06/16.
 */
public final class Folders {
    private static final Map<String, File> foldersMap = new HashMap<>();

    public static void add(String folderName, File folderFile) {
        if(!foldersMap.containsKey(folderName))
            foldersMap.put(folderName, folderFile);
    }

    public static File get(String folderName) {
        return foldersMap.get(folderName);
    }
}

File: app/src/main/java/fergaral/datetophoto/utils/Folders.kt
package fergaral.datetophoto.utils

import java.io.File
import java.util.HashMap

/**
 * Created by fer on 7/06/16.
 */
object Folders {
    private val foldersMap = HashMap<String, File>()

    fun add(folderName: String, folderFile: File) {
        if (!foldersMap.containsKey(folderName))
            foldersMap.put(folderName, folderFile)
    }

    operator fun get(folderName: String): File? {
        return foldersMap[folderName]
    }
}
